app/views/course/assessment/submissions/_guided.html.slim

```diff
@@ -0,0 +1,3 @@
+div.btn-group
```

Comment:
I'm actually going to grey out those which cannot be accessed and highlight current question ( in a separate PR ), think that is better ?